src/common/mailer/mailer.manager.ts

```diff
@@ -24,7 +25,10 @@ export class MailerManager {
     this.mailerService.sendMail({
       to: to,
       subject: subject,
-      template: 'new-order-lab',
+      template: join(
+        __dirname,
+        '../../listeners/substrate-listener/templates/new-order-lab',
```

Comment:
Wait this is not supposed to be like this. Why is a `common` module dependent on the `listeners`?